janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLKeyColumnValueStore.java

```diff
@@ -331,7 +331,7 @@ private static CreateTableWithOptions compressionOptions(final CreateTableWithOp
         int chunkLengthInKb = configuration.get(CF_COMPRESSION_BLOCK_SIZE);
 
         return createTable.withOption("compression",
-            ImmutableMap.of("sstable_compression", compressionType, "chunk_length_kb", chunkLengthInKb));
+            ImmutableMap.of("class", compressionType, "chunk_length_in_kb", chunkLengthInKb));
```

Comment:
Some of the solutions for compatibility I think:
1. We can fetch Cassandra version before creating any tables ([here](https://java-driver.docs.scylladb.com/stable/manual/core/#quick-start)): `select release_version from system.local`. I didn't check if ScyllaDB returns different versioning then Cassandra. If it returns only numbers then it's not useful too much, but if it returns something that defines it uniquely then we can determine if we are working with Cassandra 3, Cassandra 4, Cassandra 5, or ScyllaDB. In this case, if it's Cassandra and the version is >= 5 then we use new definition. Otherwise, we use previous definition.
2. We can provide an optional parameter which forces usage of one or another naming regardless of what version we are working with. If so, users can specify if they want to use `sstable_compression` or `class`. As such, they can work with Cassandra 5 or any other storage which has new names for options.

Reply:
The latest version of ScyllaDB (6.2) returns the 3.0.8 when requesting the `release_version`. Selecting the right compression options based on this information seems to be a good option.